components/gauge-circular/overview.md

```diff
@@ -1,147 +1,110 @@
 ---
 title: Overview
-page_title: Chart Overview
-description: Overview of the Chart for Blazor.
+page_title: Circular Gauge Overview
+description: Overview of the Circular Gauge for Blazor.
 slug: circular-gauge-overview
-tags: telerik,blazor,chart,overview
+tags: telerik,blazor,circular,gauge,overview
 published: True
 position: 0
 ---
 
-# Chart Overview
+# Circular Gauge Overview
 
-The <a href="https://www.telerik.com/blazor-ui/chart" target="_blank">Blazor Chart component</a> allows you to visualize data to your users in a meaningful way so they can draw conclusions. You can use a variety of chart types and control all aspects of the chart's appearance - from colors and fonts, to paddings, margins and templates.
+The Telerik Circular Gauge for Blazor represents [numerical values]({%slug circular-gauge-pointers%}) on a circular [scale]({%slug circular-gauge-scale%}).
 
-#### To use a Telerik chart for Blazor, add the `TelerikChart` tag.
+This article is separated in the following sections: 
 
->caption Basic chart with series and category axis [data binding](data-bind), and a few commonly used appearance settings
+* [Basics](#basics)
 
-````CSHTML
-Basic chart and common settings/elements
+* [Features](#features)
 
-<TelerikChart>
-	<ChartSeriesItems>
-		<ChartSeries Type="ChartSeriesType.Line" Name="Product 1 (bound to simple data)" Data="@simpleData">
-		</ChartSeries>
-		<ChartSeries Type="ChartSeriesType.Line" Name="Product 2 (bound to model)" Data="@modelData" Field="@nameof(MyDataModel.SecondSeriesValue)">
-			<ChartSeriesLabels Template="#=value# in #=dataItem.ExtraData# quarter" Visible="true"></ChartSeriesLabels>
-		</ChartSeries>
-	</ChartSeriesItems>
+* [Methods](#methods)
 
-	<ChartValueAxes>
-		<ChartValueAxis Color="red"></ChartValueAxis>
-	</ChartValueAxes>
+## Basics
 
-	<ChartCategoryAxes>
-		<ChartCategoryAxis Categories="@xAxisItems"></ChartCategoryAxis>
-	</ChartCategoryAxes>
+>caption To add a Telerik Circular Gauge for Blazor to your application:
 
-	<ChartTitle Text="Quarterly sales trend"></ChartTitle>
+1. Add the `<TelerikCircularGauge>` tag.
 
-	<ChartLegend Position="Telerik.Blazor.ChartLegendPosition.Bottom">
-	</ChartLegend>
-</TelerikChart>
+1. Add one or more instance of the `<CircularGaugePointer>` to the `<CircularGaugePointers>` collection.
 
-@code {
-	public class MyDataModel
-	{
-		public int SecondSeriesValue { get; set; }
-		public string ExtraData { get; set; }
+1. Provide a `Value` for each `<CircularGaugePointer>`.
```

Comment:
Consider enabling the center template to show the value and add that as an optional step. It will be a common real-life scenario.
Consider reducing its size a little, I think people commonly use circular gauges a bit smaller

Reply:
Done